submissions/yevheniiii/html_css_popup/css/main.css

```diff
@@ -0,0 +1,194 @@
+body {
+    font-size: 16px;
+    font-family: "Montserrat", "Arial", sans-serif;
+    font-size: 16px;
+}
+
+.nav {
+    display: block;
+    height: auto;
+}
+
+.menu {
+    margin-right: 20px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.nav-item {
+    display: flex;
+    list-style-type: none;
+    margin-left: 20px;
+}
+
+.menu img,
+.button {
+    width: 24px;
+    height: 24px;
+}
+
+.user-avatar {
+    width: 36px;
+    height: 36px;
+}
+
+.popup-triangle {
+    display: none;
+    position: absolute;
+    top: 55px;
+    right: 116px;
+    width: 0;
+    height: 0;
+    z-index: 2;
+    border-left: 12px solid transparent;
+    border-right: 12px solid transparent;
+    border-bottom: 12px solid #d4d4d4;
+}
+
+.empty {
+    position: absolute;
+    top: 2px;
+    left: -10px;
+    width: 0;
+    height: 0;
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid #fff;
+}
+
+.pop-up,
+.MoreApps {
+    position: absolute;
```

Comment:
Watch this [video ](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s)to find out how to hide checkbox in right way

Reply:
Huh, never heard about this approach. Thank you very much! @A-Ostrovnyy